data/stylish-haskell.yaml

```diff
@@ -15,6 +15,10 @@ steps:
   #     # true.
   #     add_language_pragma: true
 
+  # Format record definitions
+  - records:
+      indent: 4
```

Comment:
I think I would move the `indent` setting to the top-level, just like `columns`, so that other steps may possibly use this value?

Reply:
Good idea :-) Done
Thanks for the review!